broker/setup_broker/swift/setup_broker.sh

```diff
@@ -37,30 +38,27 @@ fi
 
 define_GCP_resources() {
     local base_name="$1"
+    local separator="$2"
```

Comment:
```suggestion
    local separator="${2:--}"
```

And then update the calls below.